src/app/components/MediaLoader/utils/buildSetting.test.ts

```diff
@@ -397,4 +400,95 @@ describe('buildSettings', () => {
       );
     });
   });
+
+  describe('OnDemandTv', () => {
+    const hindiTvBaseSettings = {
+      counterName: 'hindi.bbc_hindi_tv.tv.w172zm8920nck2z.page',
+      lang: 'hi',
+      service: 'hindi' as Services,
+      statsDestination: 'WS_NEWS_LANGUAGES',
+      producer: 'HINDI',
+      translations: hindiServiceConfig.default.translations,
+    } as BuildConfigProps;
+
+    const hindiTvMediaBlocks = hindiTvProgramme.content.blocks.map(
+      tvMediaBlock => {
+        return {
+          type: 'tvMedia',
+          model: {
+            ...tvMediaBlock,
+          },
+        };
+      },
+    );
+    it('Should process a On Demand TV block into a valid playlist item.', () => {
+      const result = buildSettings({
+        ...hindiTvBaseSettings,
+        blocks: hindiTvMediaBlocks as MediaBlock[],
+        pageType: MEDIA_PAGE,
+      });
+
+      expect(result).toStrictEqual({
+        playerConfig: {
+          product: 'news',
+          enableToucan: true,
+          appType: 'responsive',
+          autoplay: false,
+          externalEmbedUrl: '',
+          mediator: { host: 'open.test.bbc.co.uk' },
+          appName: 'news-hindi',
+          counterName: 'hindi.bbc_hindi_tv.tv.w172zm8920nck2z.page',
+          statsObject: {
+            destination: 'WS_NEWS_LANGUAGES',
+            producer: 'HINDI',
+            episodePID: 'w172zm8920nck2z',
+          },
+          ui: {
+            controls: { enabled: true },
+            fullscreen: { enabled: true },
+            locale: {
+              lang: 'hi',
+            },
+            subtitles: {
+              defaultOn: true,
+              enabled: true,
+            },
+          },
+          playlistObject: {
+            title: 'दुनिया',
+            holdingImageURL:
+              'https://ichef.bbci.co.uk/images/ic/$recipe/p0hfjjfk.png',
+            items: [
+              {
+                versionID: 'w1mskyp8ybvqtc6',
+                kind: 'programme',
+                duration: 1192,
+                vpid: 'w1mskyp8ybvqtc6',
+              },
+            ],
+            summary:
+              'ताज़ा अंतरराष्ट्रीय, क्षेत्रीय ख़बरों और विश्लेषण के लिए देखिए बीबीसी दुनिया',
+          },
+        },
+        mediaType: 'video',
+        placeholderConfig: {
+          mediaInfo: {
+            title: 'दुनिया',
+            datetime: 'PT19M52S',
+            duration: '19:52',
+            durationSpoken: 'अवधि 19,52',
+            type: 'video',
+            guidanceMessage: undefined,
+          },
+          placeholderSrc:
+            'https://ichef.bbci.co.uk/images/ic/$recipe/p0hfjjfk.png',
+          placeholderSrcset:
```

Comment:
Do these placeholder images need to be /ace/ws ones? (Let's see what the locator / origin code for the ares / clipMedia is, so that we can ensure that the values are correct)